python/mxnet/contrib/quantization.py

```diff
@@ -527,13 +527,13 @@ def quantize_model(sym, arg_params, aux_params, data_names=('data',),
 
     return qsym, qarg_params, aux_params
 
-def quantize_model_mkldnn(sym, arg_params, aux_params, data_names=('data',),
-                          ctx=cpu(), excluded_sym_names=None, excluded_op_names=None,
-                          calib_mode='entropy', calib_data=None, num_calib_batches=None,
-                          quantized_dtype='int8', quantize_mode='smart',
-                          quantize_granularity='tensor-wise', logger=None):
+def quantize_model_dnnl(sym, arg_params, aux_params, data_names=('data',),
+                        ctx=cpu(), excluded_sym_names=None, excluded_op_names=None,
+                        calib_mode='entropy', calib_data=None, num_calib_batches=None,
+                        quantized_dtype='int8', quantize_mode='smart',
+                        quantize_granularity='tensor-wise', logger=None):
     """User-level API for generating a fusion + quantized model from a FP32 model
-    w/ or w/o calibration with Intel MKL-DNN.
+    w/ or w/o calibration with Intel DNNL.
```

Comment:
```suggestion
    w/ or w/o calibration with oneDNN.
```

Reply:
@PawelGlomski-Intel - I think only names in documentation/description should be oneDNN (but I am not stick to it) and other names should be DNNL as OneDNN internally also use DNNL in runtime environment names